samples/XMasControlAddIn/AddIn/AddInInterface.al

```diff
@@ -0,0 +1,18 @@
+controladdin NavSnowControlAddIn
+{
+    RequestedHeight = 300;    
+    RequestedWidth = 300;
+    VerticalStretch = true;
+    VerticalShrink = true;
+    HorizontalStretch = true;
+    HorizontalShrink = true;
+    Scripts =         
+        'AddIn/jquery.min.js',

```

Comment:
We could use a reference directly to the CDN, for example:
http://ajax.aspnetcdn.com/ajax/jQuery/jquery-3.2.1.min.js

Then we can demonstrate another feature and don't need to keep a copy of jquery in our samples folder

Reply:
To tell you the truth I didn't knew that CDN links was supported. That's great new knowledge to me, thank. I replaced jQuery with CDN link